package.json

```diff
@@ -121,7 +121,7 @@
     "colors": "1.4.0",
     "concurrently": "8.2.2",
     "cooky-cutter": "1.5.4",
-    "cypress": "13.7.1",
+    "cypress": "13.6.4",
```

Comment:
Downgrade to workaround `Warning: vkCreateInstance: Found no drivers!` https://github.com/cypress-io/cypress/issues/29085